### DISCUSSION TITLE
Why we can not autogenerate a ObservablePropertywith a small letter

### DISCUSSION THREAD

**Discussion by milesdream** [CommunityToolkit/dotnet#991] [Ideas]
at example
  [ObservableProperty]
  [property: Category("IPTC")]
  [property: Description("City")]   
  private string _iptcCity;

should produce iptcCity

or a 

private Datetime _dateTime ;
should produce a public Dateime dateTime ;

**Comment by Sergio0694** [comment]
That is by design. Public properties with camelCase format are against the C# naming conventions.
That said, you should be able to make that work by using partial properties, with 8.4.0-preview2.
